src/components/Carousel/Carousel.css

```diff
@@ -69,4 +70,62 @@
 .carousel-item__spec-type {
     font-weight: 400;
     color: #525252;
+}
+
```

Comment:
And the design breaks between 768 and 1280 also 

![image](https://user-images.githubusercontent.com/37159177/112775998-76ad2c80-900c-11eb-870c-3109fe629195.png)